Modelling/HelperTicker.py

```diff
@@ -0,0 +1,14 @@
+import yfinance as yf
+from yahoo_fin import stock_info as si
+import pandas as pd
+
+
+def gethistoricalprice(tickers):
+    return yf.history(tickers, period="max")
+
+
+def getcurrentprice(tickers):
+    return si.get_live_price(tickers)
+
+
+# print(getcurrentprice('AMZN'))
```

Comment:
This late in the timeline, code should not contain commented out lines. They should either be removed completely or uncommented.